config/v1/types_feature.go

```diff
@@ -183,6 +183,7 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 		with(dnsNameResolver).
 		with(machineConfigNodes).
 		with(clusterAPIInstall).
+		with(installAlternateInfrastructureAWS).
```

Comment:
Given that we have two featuregates for installing with different methods, one with capi, one with the alternate approach. Do we know if they will conflict if they are enabled together? 

Should they both be disabled and enabled only in the custom featureset for now? I know Vince has a PR to drop the capi one from TP already

Reply:
I dropped this from the TechPreview feature set for this PR. I actually do think this belongs in the TechPreview feature set, but we can handle that in a separate PR, and getting the gate in will allow us to land some of the SDK work and enable more QE testing.

Following up in a separate comment to answer your question...

Reply:
> Do we know if they will conflict if they are enabled together?

The way I have been thinking about this for the installer is that we would essentially have precedence. Perhaps a pseudocode illustratipn is helpful:

```
if fg.Enabled(capiInstall) {
     return capiInstall
}

if fg.Enabled(sdkInstall) {
     return sdkInstall
}

return terraformInstall
```

So having both enabled, capi would conflict with SDK in that it overrides SDK, but SDK would not break capi if they are both enabled.

CAPI install is not stable enough to be in the tech preview feature set. AWS SDK install is pretty stable, and I think we should consider putting it in the techpreview feature set, because I believe there is some motivation to roll it out as the default AWS install experience for all users. Happy to discuss more subsequently.

Reply:
Do you think it would make sense initially to add validation that does not allow them to be added simultaneously? Eventually we would have to remove this but it might help prevent someone accidentally trying to enable both and thinking that they got an SDK install when actually they got a CAPI install?